R/sha1.R

```diff
@@ -0,0 +1,177 @@
+# functions written by Thierry Onkelinx
+sha1 <- function(x, digits = 14, zapsmall = 7){
+    UseMethod("sha1")
+}
+
+sha1.default <- function(x, digits = 14, zapsmall = 7) {
+    stop("sha1() has not method for the '", class(x), "' class")
```

Comment:
Minor rewording:  _has no method_.  But we get that next time.

Also, I may consider adding `call.=FALSE` -- but I don't do this yet in other places in digest so no rush.